bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -51,6 +51,8 @@ class ProjectBuilder {
                 buildCmd = ':app:bundleRelease';
             } else if (cmd === 'debug') {
                 buildCmd = ':app:bundleDebug';
+            } else {
```

Comment:
What about assigning the `cmd` value already to `buildCmd` in line 49? (i.e., `let buildCmd = cmd;`) This would remove the else block and only overwrite the `buildCmd` in the necessary debug and release cases.

Reply:
No worries, take your time

Reply:
@timbru31 I've made the commit.